compiler/locoex-customop/README.md

```diff
@@ -1,9 +1,9 @@
 # locoex
 
-_locoex_ is an extention of loco. Classes with `COp` prefix enables *Custom Operation*.
+_locoex_ is an extension of loco. Classes with the `COp` prefix enable *Custom Operation*.
 In this version, a *custom operation* means one of the following:
 
-1. an op that is supported by Tensorflow but not supported both by the moco and the onert
```

Comment:
"not supported both" is still unclear. What if only one of them supports the op?

Reply:
I think `both` is not that important. either can benefit from this module.

Reply:
Do moco and onert support same ops? If not, it's either:

    1. an op that is supported by Tensorflow but not supported by moco or onert

or

    1. an op that is supported by Tensorflow but not supported by both moco and onert

and these mean different things.

Reply:
I think meaning of this is CustomOp is for Ops that are not supported from existing Ops (from schema) that we have to support from TensorFlow models.
For this, I think it is the first one.
As I understand, the existing writing using `and` is used for (both moco and onert) can "benefit" from using CustomOp.